docs/markdown/Fs-module.md

```diff
@@ -29,3 +31,71 @@ name exists on the file system. This method follows symbolic links.
 
 Takes a single string argument and returns true if the path pointed to
 by the string is a symbolic link.
+
+## File Parameters
+
+### hash
+
+The `fs.hash(filename)` method computes the requested hash sum of a file.
+The available hash methods include: md5, sha1, sha224, sha256, sha384, sha512.
```

Comment:
This should specify the exact format. Is it commonly called just "hexdigest" or is that a pythonism?

Reply:
Oops I forgot to include the second argument in the docs here. I mean to have the digest as a required user option.

The exact hex digests available depend on the particular Python install, so I only listed the ones virtually always available.